### DISCUSSION TITLE
FPS counter

### DISCUSSION THREAD

**Discussion by krishunt** [R2Northstar/Northstar#457] [Q&A]
How do I get the FPS to display onscreen with Northstar? Northstar doesn't show up in the EA app, so I don't know where to enter the `+cl_showfps 1` command.

**Comment by ASpoonPlaysGames** [comment]
You can either add it to `ns_startup_args` (found in your titanfall folder), or you can just type the command `cl_showfps 1` into the console in game (console can be opened using the \` key by default)
